README.md

```diff
@@ -64,6 +64,9 @@ steps:
 | `true`       | `foo` = `{ "bar": "baz" }`<br>`ham` = `eggs`     | `FOO_BAR` = `baz` AND<br>`ham` = `eggs`             | If multiple secrets, values that can be parsed into a JSON will be parsed and flattened |
 | `false`      | `dev_foo` = `{ "bar": "baz" }`                   | `DEV_FOO` = `{ "bar": "baz" }`                      | Not parsed                                                                              |
 
+- `disable-warnings`
+  - If `disable-warnings: true`. the annotation warnings regarding posix compliance in GitHub Actions output will be supressed.
```

Comment:
```suggestion
  - If `disable-warnings: true`, the annotation warnings regarding POSIX compliance in GitHub Actions output will be supressed.
```

Reply:
Also, what's an `annotation warning`? Does it mean anything different from a simple `warning`? Can we simplify it in the below way?
```suggestion
  - If `disable-warnings: true`, warnings regarding POSIX compliance in GitHub Actions output will be suppressed.
```